mod-record-specifications-server/src/main/java/org/folio/rspec/integration/kafka/EventProducer.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.rspec.integration.kafka;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.tools.kafka.KafkaUtils;
+import org.springframework.kafka.core.KafkaTemplate;
+
+@Log4j2
+@RequiredArgsConstructor
+public abstract class EventProducer<T, D> {
+
+  private final KafkaTemplate<String, D> template;
+  private final FolioExecutionContext context;
+
+  public void sendMessage(T data) {
+    log.info("Sending events to Kafka [topic: {}]", topicName());
```

Comment:
Maybe `Sending message` as a method is `sendMessage`
or even better to follow  format that already used in the module, it's probably smth like:
```suggestion
    log.info("sendMessage::...[topic={}]", topicName());
```

Reply:
Maybe also rename EventProducer to MessageProducer or vice versa, rename method 

Let's define consistency in this class :)